### PR TITLE
Project README update

### PR DESCRIPTION
**Pull request by tsegismont**
An update to reflect the current state of the project and highlight goals.

### CODE REVIEW

**Comment by tsegismont** [comment]
Can anyone please review this doc update?

For the JAX-RS module README, I believe we should remove endpoint documentation, which is already published on the H* website, and keep simple JSON samples only (like POST metric data and read metric data).

File: README.adoc [hunk 1]
```diff
@@ -8,66 +8,98 @@ image::https://travis-ci.org/hawkular/hawkular-metrics.svg?branch=master[Build S
...
-= Project to implement a MaaS and ChaaS
+== About
 
-* MaaS: Metrics as a Service
-* ChaaS: Charts as a Service
+Hawkular Metrics is the metric data store for the http://www.hawkular.org/[Hawkular] project.
+It can also be used independently.
 
-== Quickstart
+It uses https://cassandra.apache.org/[Apache Cassandra] as a backend and is comprised of:
```
Comment (jsanda) [inline README.adoc]: Might be good to mention here that we require Cassandra 2.1.2 or later.

**Comment by tsegismont** [comment]
@jsanda thx for the review. I've pushed a commit to emphasize the dependency on C* 2.1.x at the beginning of the file ( b70cf99 ).

**Comment by tsegismont** [comment]
Thanks for reviewing and merging. Any opinion on the JAX-RS module README question?

**Comment by jsanda** [comment]
I think it looks good. I think it might make sense to eventually try to move most the JAX-RS docs to the hawkular site as well, particularly the request/response examples. We need to keep up with docs so it is good to see these changes. Thanks!

**Pull request merged**